js/index.js

```diff
@@ -1,7 +1,80 @@
 // Iteration 1: Names and Input
-
+let hacker1 = "Sophie";
+console.log(`The driver is ${hacker1}`);
+let hacker2 = "Ceyda";
+console.log(`The navigator is ${hacker2}`);
 
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if (hacker1.length < hacker2.length) {
+  console.log(`The navigator has the longest name, it has ${hacker2.length} characters.`)
+} else {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
+
+// Iteration 3: Loops
+// 3.1
+var nameSeparatedBySpace = "";
+for (let i = 0; i < hacker1.length; i++) {
+  nameSeparatedBySpace += hacker1[i].toUpperCase() + " ";
+}
+console.log(nameSeparatedBySpace);
+
+//3.2
+var navigatorNameReversed = "";
+for (let i = hacker2.length-1; i >= 0; i--) {
+  navigatorNameReversed += hacker2[i];
+}
+console.log(navigatorNameReversed)
+
+//3.3
+if (hacker1.localeCompare(hacker2) === 1) {
```

Comment:
Please check out the MDN documentation on localCompare. The output is not necessarily 1, 0 or -1. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare